css-scroll-anchoring-1/Overview.bs

```diff
@@ -85,16 +86,41 @@ An anchor node can be any <a>box</a> except one for a non-<a>atomic inline</a>.
 The anchor node is always a <a>descendant</a> of the <a>scrolling box</a>.
 In some cases, a scrolling box may not select any anchor node.
 
+An element |C| is a <dfn id="anchor-viable-candidate">viable candidate</dfn>
+for becoming a scroll anchor for a scrolling box |S| if it meets all of the
+following criteria:
+  * |C| is a non-<a>atomic inline</a>.
+  * |C| is <a>partially visible</a> or <a>fully visible</a> in |S|
+  * |C| is a descendant of |S|
+  * |C| is not in an <a>excluded subtree</a>
+  * None of the ancestors of |C| up to |S| are in an <a>excluded subtree</a>
+
+Some elements are considered to be <dfn
+id="anchor-priority-candidates">priority candidates</dfn> for anchor selection:
+  1. An element that <a href="https://html.spec.whatwg.org/multipage/semantics-other.html#selector-focus">has focus</a>.
+  2. An element containing the <a href="https://drafts.csswg.org/css-ui-3/#insertion-caret">insertion caret</a>.
+  3. An element containing the current active selected match of the
```

Comment:
This is a bit ill-defined, IMO. What does a current active selected match mean? Firefox allows you to highlight all occurrences, is only the first one active? We should probably have a proper spec for find-in-page if we're making web-observable behavior depend on it.

Reply:
You're right that it would be great to have the find-in-page algorithm specified somewhere.. I don't know if that's something that can be done easily though since we need to agree on the details to match all of the browsers.

For your specific questions here, although Firefox, Safari, and Chrome all highlight (or are capable of highlighting) all matches, the active one is usually highlighted differently. By active I mean the one that is brought into view if it's not in view yet. This is the match that advances when the user clicks 'next' or 'previous' in the search dialog.

In lieu of a proper spec for find-in-page, I would appreciate any thoughts on how this could be phrased better...

Reply:
Wording looks ok, it's just very fishy to reference something that is not specified in a spec like this.

Reply:
Yeah, I agree. I think it's worthwhile for us to at least file an issue to specify the find-in-page algorithm in some way, since it is a common enough operation